tests/debugger/test_debugger_pii.py

```diff
@@ -111,16 +132,36 @@ def filter(keys_to_filter):
 @features.debugger_pii_redaction
 @scenarios.debugger_pii_redaction
 class Test_Debugger_PII_Redaction(base._Base_Debugger_Test):
+    probes_state = None
+
     def _setup(self):
+
         self.expected_probe_ids = [
             "log170aa-acda-4453-9111-1478a6method",
         ]
 
-        interfaces.library.wait_for_remote_config_request()
-        interfaces.agent.wait_for(self.wait_for_all_probes_installed, timeout=30)
-        self.weblog_responses = [weblog.get("/debugger/pii")]
+        # trick to avoid to re-run the setup function for each test: we save the result of weblog_responses/probes_state
+        # in class rather than class instance (each test got it's own class instance)
+        # and if it exists, we pick it from there
+        if Test_Debugger_PII_Redaction.probes_state is None:
+            # setup has not been run yet
+            # rc.send_command(raw_payload=rc.build_debugger_command(probes=None, version=0))
```

Comment:
@shurivich

The current API does not know how to check that an empty command has been applied, but is it really needed ( see comment above in the deleted file)

Reply:
Got it, nice.